tests/test.py

```diff
@@ -6173,6 +6165,58 @@ def testServerRead(self):
         super().testServerRead()
         self.assertIn("Finished processing client's connection", self.read_log(self.acra))
 
+class TestAcraIgnoresLegacyKeys(AcraCatchLogsMixin, BaseTestCase):
+    """
+    Ensure AcraServer won't exit with error when started with flags for key caching,
+    while keystore contains legacy keys (Connector<->Server, Connector<->Translator)
+    """
+
+    legacy_key_files = [
+        'testclientid',
+        'testclientid.pub',
+        'testclientid_server',
+        'testclientid_server.pub',
+        'testclientid_translator',
+        'testclientid_translator.pub',
+    ]
+
+    def checkSkip(self):
+        if KEYSTORE_VERSION != 'v1':
+            self.skipTest("test only for keystore v1")
+
+    def setUp(self):
+        super().setUp()
+
+        def create_key_file(name):
+            open(f"{KEYS_FOLDER.name}/{name}", "w").close()
+
+        try:
+            for key_file in self.legacy_key_files:
+                open(f"{KEYS_FOLDER.name}/{key_file}", "w").close()
+        except:
+            self.tearDown()
+            raise
+
+    def tearDown(self):
+        for key_file in self.legacy_key_files:
+            os.remove(f"{KEYS_FOLDER.name}/{key_file}")
```

Comment:
```
$ python3 -c 'import os; os.remove("/tlqkeqwleqwlekqweq");'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
FileNotFoundError: [Errno 2] No such file or directory: '/tlqkeqwleqwlekqweq'
```

we should catch exceptions here too or we may fail on first file and leave all remaining files

Reply:
Sure, wrapped with `try: ... except: pass`